apps/web/components/question/question-types/QuestionMissing.tsx

```diff
@@ -0,0 +1,4 @@
+export default function QuestionMissing() {
+  // TODO - better error screen
```

Comment:
Is there a ticket for this?

Reply:
We don't currently have any designs for them. 

[This ticket](https://hous-bssb.atlassian.net/browse/HOUSNAV-66) exists in our backlog, but we probably need to make a design ticket first.

Reply:
Created a low priority design ticket for this: https://hous-bssb.atlassian.net/browse/HOUSNAV-70